web/assets/auto-resize-text-area.js

```diff
@@ -0,0 +1,13 @@
+const autoResizeTextArea = () => {
+    const txtarea = document.getElementsByTagName("textarea");
+    for (let i = 0; i < txtarea.length; i++) {
+        txtarea[i].setAttribute("style", "height:" + (txtarea[i].scrollHeight) + "px;overflow-y:hidden;");
+        txtarea[i].addEventListener("input", OnInput, false);
+    }
+
+    function OnInput() {
+        this.style.height = "auto";
```

Comment:
auto resets the height of the text area before the changes are applied with scrollHeight